react/containers/DeliveryContainer.js

```diff
@@ -88,7 +91,9 @@ class DeliveryContainer extends Component {
       onChangeAddress,
       rules,
       disableCalculateButton,
+      validateNumber,
     } = this.props
+    validateNumber(rules)
```

Comment:
How does the application ensure that the `validateNumber` used is not the imported function but the prop?

In this case it would be interesting to use like this, wouldn't it?
```suggestion
    this.props.validateNumber(rules)
```
or change prop name...